core/src/client/rpc.rs

```diff
@@ -124,7 +124,7 @@ trait EthRpc<TX: TransactionResponse + RpcObject, TXR: RpcObject, R: ReceiptResp
     #[method(name = "getLogs")]
     async fn get_logs(&self, filter: Filter) -> Result<Vec<Log>, ErrorObjectOwned>;
     #[method(name = "getFilterChanges")]
-    async fn get_filter_changes(&self, filter_id: U256) -> Result<Vec<Log>, ErrorObjectOwned>;
+    async fn get_filter_changes(&self, filter_id: U256) -> Result<FilterChanges, ErrorObjectOwned>;
```

Comment:
Should we also add the newPendingTransactionFilter and newBlockFilter methods to rpc?

Reply:
Those two are already part of the RPC. 

https://github.com/eshaan7/helios/blob/master/core/src/client/rpc.rs#L134-L137

Reply:
The methods are there in the RPC but their corresponding Wasm bindings are missing, for which I had already created an issue (https://github.com/a16z/helios/issues/451).